glue/viewers/profile/state.py

```diff
@@ -152,12 +152,12 @@ def _reference_data_changed(self, *args):
                     self.x_att_helper.set_multiple_data([])
                 else:
                     self.x_att_helper.set_multiple_data([self.reference_data])
-                    if self._display_world:
-                        self.x_att_helper.world_coord = True
-                        self.x_att = self.reference_data.world_component_ids[0]
-                    else:
-                        self.x_att_helper.world_coord = False
-                        self.x_att = self.reference_data.pixel_component_ids[0]
+                    # if self._display_world:
+                    #     self.x_att_helper.world_coord = True
+                    #     self.x_att = self.reference_data.world_component_ids[0]
+                    # else:
```

Comment:
Remove if unused?

Reply:
Yup, was hesitant before because I wasn't sure if I was in the right direction.